specification/compute/resource-manager/Microsoft.Compute/stable/2017-09-01/examples/ListAvailableResourceSkus.json

```diff
@@ -0,0 +1,170 @@
+                                                    "name":  "MemoryGB",
+                                                    "value":  "0.75"
+                                                },
+                                                {
+                                                    "name":  "MaxDataDiskCount",
+                                                    "value":  "1"
+                                                },
+                                                {
+                                                    "name":  "LowPriorityCapable",
+                                                    "value":  "False"
+                                                },
+                                                {
+                                                    "name":  "PremiumIO",
+                                                    "value":  "False"
+                                                },
+                                                {
+                                                    "name":  "vCPUsAvailable",
+                                                    "value":  "1"
+                                                },
+                                                {
+                                                    "name":  "ACUs",
+                                                    "value":  "50"
+                                                },
+                                                {
+                                                    "name":  "vCPUsPerCore",
+                                                    "value":  "1"
+                                                },
+                                                {
+                                                    "name":  "EphemeralOSDiskSupported",
+                                                    "value":  "False"
+                                                },
+                                                {
+                                                    "name":  "AcceleratedNetworkingEnabled",
+                                                    "value":  "False"
+                                                },
+                                                {
+                                                    "name":  "RdmaEnabled",
+                                                    "value":  "False"
+                                                },
+                                                {
+                                                    "name":  "MaxNetworkInterfaces",
+                                                    "value":  "2"
+                                                }
+                                            ],
+                           "locationInfo":  [
+                                                {
+                                                    "location":  "westus"
```

Comment:
Pls. add zone details or data in the example too.

Reply:
Updated with zone details and zone capabilities